### PR TITLE
fix broken link

### PR DESCRIPTION
**Pull request by cm-wada-yusuke**
I fixed a broken user manual link, in README.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@ Dan Allen <https://github.com/mojavelinux[@mojavelinux]>; Guillaume Grossetie <h
 ifndef::uri-rel-file-base[:uri-rel-file-base: link:]
 :uri-contribute: {uri-rel-file-base}CONTRIBUTING.adoc
 :uri-contribute-code: {uri-rel-file-base}CONTRIBUTING-CODE.adoc
-:uri-user-manual: https://asciidoctor-docs.netlify.com/asciidoctor.js/3.0
+:uri-user-manual: https://asciidoctor-docs.netlify.com/asciidoctor.js/latest
```
Comment (Mogztter) [inline README.adoc:12]: Thanks!
Now that the new documentation site is live, we should use: 

```suggestion
:uri-user-manual: https://docs.asciidoctor.org/asciidoctor.js/latest/
```

**Review by Mogztter — CHANGES_REQUESTED** [review]
Thanks!
I left a comment to replace the "internal" DNS (asciidoctor-docs.netlify.com) with the official one (docs.asciidoctor.org) now that the documentation site is live.

**Comment by Mogztter** [comment]
Perfect 👌

File: README.adoc (hunk 1 above)
Comment (cm-wada-yusuke) [reply README.adoc]: Oh, thanks. My commit is evolved 😄

**Comment by cm-wada-yusuke** [comment]
Thanks for everything!

**Pull request merged**